src/_components/AxisX.percent-range.html.svelte

```diff
@@ -24,10 +24,10 @@
 	/** @type {Boolean} [snapLabels=false] - Instead of centering the text labels on the first and the last items, align them to the edges of the chart. */
 	export let snapLabels = false;
 
-	/** @type {Function} [format=d => d] - A function that passes the current tick value and expects a nicely formatted value in return. */
+	/** @type {(d: any) => string} [format=d => d] - A function that passes the current tick value and expects a nicely formatted value in return. */
```

Comment:
Maybe `any` could be more precisely defined.

Reply:
It's probably o.k. to keep these values as `any` then, I've also changed the other places of `Array<Number>` to `Array<any>` since we can't really know what users might use in their scale functions.